crates/bevy_gltf/src/loader.rs

```diff
@@ -502,7 +502,7 @@ async fn load_gltf<'a, 'b, 'c>(
                 bevy_utils::tracing::debug!(
                     "Automatically calculating missing vertex normals for geometry."
                 );
-                mesh.compute_flat_normals();
+                mesh.compute_normals();
```

Comment:
This is what we had in 0.13. It's not ideal to duplicate vertices like that, but at least it didn't crash and the normals were always flat. We could definitely add a check to not always duplicate vertices.

```suggestion
                let vertex_count_before = mesh.count_vertices();
                mesh.duplicate_vertices();
                mesh.compute_flat_normals();
                let vertex_count_after = mesh.count_vertices();

                if vertex_count_before != vertex_count_after {
                    bevy_log::debug!("Missing vertex normals in indexed geometry, computing them as flat. Vertex count increased from {} to {}", vertex_count_before, vertex_count_after);
                } else {
                    bevy_log::debug!(
                        "Missing vertex normals in indexed geometry, computing them as flat."
                    );
                }
```